store/searchlayer/user_layer.go

```diff
@@ -142,6 +145,10 @@ func (s *SearchUserStore) autocompleteUsersInChannelByEngine(engine searchengine
 	return autocomplete, nil
 }
 
+func (s *SearchUserStore) sanitizeTerm(term string) string {
+	return strings.TrimLeft(term, "@")
+}
```

Comment:
nit: if we're not using the store as part of this function, we can move it to an external file like `utils.go` in the same package to be an independent method

Reply:
Good point. On it

Reply:
Done